ethereum-consensus/src/state_transition/context.rs

```diff
@@ -262,6 +262,14 @@ impl Context {
         Self::from(phase0_preset, altair_preset, bellatrix_preset, config)
     }
 
+    pub fn for_holesky() -> Self {
+        let config = &configs::holesky::config();
+        let phase0_preset = &phase0::minimal::PRESET;
+        let altair_preset = &altair::minimal::PRESET;
+        let bellatrix_preset = &bellatrix::minimal::PRESET;
```

Comment:
refer: https://github.com/eth-clients/holesky/blob/main/custom_config_data/config.yaml#L2

```suggestion
        let phase0_preset = &phase0::mainnet::PRESET;
        let altair_preset = &altair::mainnet::PRESET;
        let bellatrix_preset = &bellatrix::mainnet::PRESET;
```